<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewDroneItem</class>
 <widget class="QWidget" name="OverviewDroneItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>627</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerBar">
     <item>
      <widget class="QLabel" name="header">
       <property name="text">
        <string>Drone #??</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="droneDetails">
     <item row="0" column="0">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="statusValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="batteryLabel">
       <property name="text">
        <string>Battery:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="batteryValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="connectivityLabel">
       <property name="text">
        <string>Connectivity:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="connectivityValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="searchAreaLabel">
       <property name="text">
        <string>Searched area:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="searchAreaValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="locatedPeopleLabel">
       <property name="text">
        <string>Located people:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="locatedPeopleValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="emergencyButton">
       <property name="toolTip">
        <string>Emergency Landing</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../../../Desktop/shae/workstation/src/application.qrc">
         <normaloff>:/ui/icons/emergency</normaloff>:/ui/icons/emergency</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="returnHomeButton">
       <property name="toolTip">
        <string>Go Home</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../../../Desktop/shae/workstation/src/application.qrc">
         <normaloff>:/ui/icons/home</normaloff>:/ui/icons/home</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="haltButton">
       <property name="toolTip">
        <string>Halt</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../../../Desktop/shae/workstation/src/application.qrc">
         <normaloff>:/ui/icons/halt</normaloff>:/ui/icons/halt</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../../../Desktop/shae/workstation/src/application.qrc"/>
 </resources>
 <connections/>
</ui>
